docker/django/Dockerfile

```
@@ -0,0 +1,38 @@
+# ==============================
+FROM python:3.7 as base
+# ==============================
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN useradd -Ums /bin/bash -d /appuser appuser
+
+WORKDIR /app
+
+COPY --chown=appuser:appuser ../../requirements.txt .
```

Not 100% sure about this, but I think since the project root is usually the "context" directory you should be able to write this like (and from what I gather it seems to be a common practice).
```suggestion
COPY --chown=appuser:appuser requirements.txt .
```

---

Yeah this and the other suggestions seem to work just as well. Changing these...